src/js/components/Accordion.js

```diff
@@ -63,7 +70,12 @@ export default class Accordion extends Component {
 
 Accordion.propTypes = {
   animate: PropTypes.bool,
+  disabled: PropTypes.bool,
   openMulti: PropTypes.bool,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
```

Comment:
What's the rationale for having a custom icon? I would think wanting to keep controls consistent across apps and sites so that user's aren't required to accommodate differences would call for not allowing custom icons.